library/src-scala3/scala/quoted/package.scala

```diff
@@ -0,0 +1,18 @@
+package scala
+
+package object quoted {
+
+  implicit class LiftExprOps[T](val x: T) extends AnyVal {
+    def toExpr(implicit ev: Liftable[T]): Expr[T] = ev.toExpr(x)
+  }
+
+  implicit class ListOfExprOps[T](val list: List[Expr[T]]) extends AnyVal {
```

Comment:
Don't you have this conversion for free if you make `Liftable[List[T]]` implicitly available?

Reply:
No, `Liftable[List[T]]` transforms `List[T]` into `Expr[List[T]]`. But here we transform `List[Expr[T]]` into `Expr[List[T]]`.

Reply:
This is a generalisation of your [`liftListOfAny`](
https://github.com/lampepfl/dotty/pull/5543/files#diff-c03a5fbe7d1df314004a2fceb5b6df73L59).